docs/readthedocs/source/doc/Orca/Tutorial/k8s.md

```diff
@@ -130,21 +130,28 @@ def train_data_creator(config, batch_size):
     return trainloader
 ```
 
-
 ---
-## 2. Create BigDL K8s Container 
-### 2.1 Pull Docker Image
-Please pull the BigDL [`bigdl-k8s`](https://hub.docker.com/r/intelanalytics/bigdl-k8s/tags) image (built on top of Spark 3.1.3) from Docker Hub as follows:
+### 2 Pull Docker Image
```

Comment:
Separate this part since for yaml file, users don't necessarily need to create a client container.